src/test/java/ebl/config/TestConfig.java

```diff
@@ -17,6 +18,8 @@
 
   // API endpoints
   public static final String SHIPPING_INSTRUCTIONS = "/v2/shipping-instructions";
+  public static final String TRANSPORT_DOCUMENTS = "/v2/transport-documents";
+  public static final String TRANSPORT_DOCUMENT_SUMMARIES =   "/v2/transport-document-summaries";
```

Comment:
Seems like the last dev merge added a duplicate for some reason

Reply:
Yes, just saw that 👍